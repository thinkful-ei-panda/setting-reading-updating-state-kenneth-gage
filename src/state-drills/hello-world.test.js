import React from 'react';
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import HelloWorld from './hello-world';

describe ( `helloWorld Component`, () => {
	
	it ( 'renders without crashing', () => {
	  
		const div = document.createElement ( 'div' );
		
		ReactDOM.render ( <HelloWorld />, div );
		
		ReactDOM.unmountComponentAtNode ( div );

	})

	it ( 'renders the UI as expected', () => {

	  expect(

		renderer.create ( <HelloWorld /> ).toJSON ()

	  ).toMatchSnapshot ()

	})

})