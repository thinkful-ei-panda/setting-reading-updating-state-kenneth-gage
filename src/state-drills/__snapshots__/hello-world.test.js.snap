// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helloWorld Component renders the UI as expected 1`] = `
<div>
  <p>
    Hello 
    World
  </p>
  <button
    onClick={[Function]}
  >
    Friend
  </button>
  <button
    onClick={[Function]}
  >
    React
  </button>
  <button
    onClick={[Function]}
  >
    World
  </button>
</div>
`;
